mmdet/models/roi_heads/cascade_roi_head.py

```diff
@@ -505,3 +504,74 @@ def aug_test(self, features, proposal_list, img_metas, rescale=False):
             return [(bbox_result, segm_result)]
         else:
             return [bbox_result]
+
+    def onnx_export(self, x, proposals, img_metas):
+
+        assert self.with_bbox, 'Bbox head must be implemented.'
+        assert proposals.shape[0] == 1, 'Only support one input image ' \
+                                        'while in exporting to ONNX'
+        # remove the scores
+        rois = proposals[..., :-1]
+        batch_size = rois.shape[0]
+        num_proposals_per_img = rois.shape[1]
+        # Eliminate the batch dimension
+        rois = rois.view(-1, 4)
+
+        # add dummy batch index
+        rois = torch.cat([rois.new_zeros(len(rois), 1), rois], dim=-1)
```

Comment:
if `rois.shape` would change, better to use rois.shape[0] instead of `len(rois)`.